tests/publish-instance.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test } from "./fixtures/lxd-test";
+import { deleteAllImages } from "./helpers/images";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitInstance,
+} from "./helpers/instances";
+
+let instance = randomInstanceName();
+
+test.beforeAll(async ({ browser, browserName }) => {
```

Comment:
there is no need to use `beforeAll` and `afterAll` hooks. Since there is only one test, you can just add the logic in the test directly